test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -223,6 +235,10 @@ func (op *op) UnmarshalJSON(b []byte) error {
 		&createNamespacesOp{},
 		&createPodsOp{},
 		&createPodSetsOp{},
+		&createResourceClaimsOp{},
+		&createResourceClaimTemplateOp{},
+		&createResourceClassOp{},
+		&createResourceDriverOp{},
```

Comment:
Any chance these can all be replaced with a single createObjectsFromTemplate?

Reply:
I could use Go generics?